cljfmt/test_resources/cljfmt.edn

```diff
@@ -0,0 +1,4 @@
+{:extra-indents {com.foo/a [[:inner 0]]
+                 com.unrelated/a [[:inner 0]]}
+ :extra-configs ["extra1-cljfmt.edn"
+                 "extra2-cljfmt.edn"]}
```

Comment:
These resources should be put under a `cljfmt` folder, rather than at the root, to better group them and prevent any namespace clashes.

Reply:
sounds good, can do

Reply:
I put them under `cljfmt/config_test`, to further ns them. Can change it to just cljfmt if that's not acceptable